scipy/stats/tests/test_stats.py

```diff
@@ -1655,11 +1655,25 @@ def test_theilslopes():
     assert_almost_equal(slope, 0.5)
     assert_almost_equal(intercept, 0.5)
 
+    slope, intercept, lower, upper = stats.theilslopes([0, 1, 1],
```

Comment:
one extra test case to be added that checks that an error is raised for invalid method like xyz => there should be some examples in test_stats.py